### ISSUE TITLE
【Beg for help】for the information of 【package denpendencies of the IonML code】

### ISSUE THREAD

**Issue by superXiWang** [CodingWZL/IonML#1]
Hello, zhilong, it is my pleasure to encounter your paper "IonML: A physically inspired machine learning platform to directed design
superionic conductors". It is a remarkable work and I want to follow.
【Problem description】
However, I have trouble with making the code run. I have tried to set up my python runing envionment using "conda" with a version of Python 3.8 and have installed all the package. But it still can NOT work. 
【What I ask for】
So I wonder, if you can kindly export a copy of the environment in which the code can be run, including the dependency packages and corresponding versions? 
If you use conda too, just a simple scipt can work out.  "conda env export > environment.yaml"
So could you please send this envionment.yaml file to my email (xiwang_personal@163.com) ? This will help a lot!
Thank you most sincerely!

**Comment by CodingWZL** [comment]
Hello, thanks for your interest, below we have provided the relevant dependencies (which will also be communicated via email) that we hope will help you. If you have any questions, please feel free to contact us.
--------------
name: my_ionml
channels:
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge/
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/msys2/
  - https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/free/
  - defaults
  - conda-forge
dependencies:
  - _libgcc_mutex=0.1=main
  - _openmp_mutex=4.5=1_gnu
  - ca-certificates=2021.10.26=h06a4308_2
  - certifi=2020.6.20=pyhd3eb1b0_3
  - ld_impl_linux-64=2.35.1=h7274673_9
  - libffi=3.3=he6710b0_2
  - libgcc-ng=9.3.0=h5101ec6_17
  - libgomp=9.3.0=h5101ec6_17
  - libstdcxx-ng=9.3.0=hd4cf53a_17
  - ncurses=6.3=h7f8727e_2
  - openssl=1.1.1m=h7f8727e_0
  - pip=21.2.2=py36h06a4308_0
  - python=3.6.13=h12debd9_1
  - readline=8.1.2=h7f8727e_1
  - setuptools=58.0.4=py36h06a4308_0
  - sqlite=3.37.0=hc218d9a_0
  - tk=8.6.11=h1ccaba5_0
  - wheel=0.37.1=pyhd3eb1b0_0
  - xz=5.2.5=h7b6447c_0
  - zlib=1.2.11=h7f8727e_4
  - pip:
    - ase==3.22.1
    - attrs==21.4.0
    - bayes==0.1.1
    - bayesian-optimization==1.2.0
    - charset-normalizer==2.0.10
    - colorama==0.4.4
    - cycler==0.11.0
    - cython==0.29.27
    - dataclasses==0.8
    - decorator==4.4.2
    - dscribe==1.2.2
    - future==0.18.2
    - greenlet==1.1.2
    - idna==3.3
    - importlib-metadata==4.8.3
    - importlib-resources==5.4.0
    - joblib==1.1.0
    - jsonschema==3.2.0
    - kiwisolver==1.3.1
    - llvmlite==0.36.0
    - matminer==0.6.5
    - matplotlib==3.3.4
    - mendeleev==0.9.0
    - monty==2022.1.19
    - mpmath==1.2.1
    - networkx==2.5.1
    - numba==0.53.1
    - numpy==1.19.5
    - packaging==21.3
    - palettable==3.3.0
    - pandas==1.1.5
    - pillow==8.4.0
    - pint==0.17
    - plotly==5.5.0
    - pybind11==2.10.1
    - pyfiglet==0.8.post1
    - pygments==2.11.2
    - pymatgen==2020.10.20
    - pymongo==4.0.1
    - pyparsing==3.0.7
    - pyrsistent==0.18.0
    - python-dateutil==2.8.2
    - pytz==2021.3
    - requests==2.27.1
    - ruamel-yaml==0.17.20
    - ruamel-yaml-clib==0.2.6
    - scikit-learn==0.24.2
    - scipy==1.5.4
    - six==1.16.0
    - sparse==0.13.0
    - spglib==1.16.3
    - sqlalchemy==1.4.31
    - sympy==1.9
    - tabulate==0.8.9
    - tenacity==8.0.1
    - threadpoolctl==3.0.0
    - tqdm==4.62.3
    - typing-extensions==4.0.1
    - uncertainties==3.1.6
    - urllib3==1.26.8
    - xgboost==1.1.1
    - zipp==3.6.0
prefix: /home/wzl/Software/anaconda3/envs/my_ionml
